content/docs/tasks/security/authz-tcp/index.md

```diff
@@ -185,27 +179,25 @@ to access the mongoDB service.
 
     Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`).  You should see:
 
-    * The "Book Details" section in the lower left part of the page, including type, pages, publisher, etc.
-    * The "Book Reviews" section in the lower right part of the page with red stars again.
+    * The "Book Details" section on the lower left of the page includes book type, number of pages, publisher, etc.
+    * The "Book Reviews" section on the lower right of the page includes red stars.
 
     > There may be some delays due to caching and other propagation overhead.
 
-1. To confirm the mongoDB service can only be accessed by service account `bookinfo-ratings-v2`:
+1. To confirm the MongoDB service can only be accessed by service account `bookinfo-ratings-v2`:
 
     Run the following command to re-deploy the `v2` of `ratings` with service account `default`:
 
     {{< text bash >}}
     $ kubectl delete -f @samples/bookinfo/platform/kube/rbac/ratings-v2-add-serviceaccount.yaml@
     $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo-ratings-v2.yaml@
-    serviceaccount "bookinfo-ratings-v2" deleted
-    deployment "ratings-v2" deleted
-    deployment "ratings-v2" created
     {{< /text >}}
 
     Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`).  You should see:
 
-    * The "Book Details" section in the lower left part of the page, including type, pages, publisher, etc.
-    * The "Book Reviews" section in the lower right part of the page with "Ratings service is currently unavailable".
+    * The "Book Details" section on the lower left of the page includes book type, number of pages, publisher, etc.
```

Comment:
I feel like I edited these texts before and yet you add them unchanged saved for the preposition. I did change more than the prepositions if I recollect correctly.

Reply:
I can change `Book Details` to bold (here and other places) but I still think the page doesn't look good with so many bold elements.